mrc-ide/packit
# mrc-4372 display downloadable files

PR displays downloadable files. 

Step to test
* From packet explorer, click on a packet
* then additional sidebar items will appear, click on downloads.

Each download heading, where does the data come from ?

Endpoint returns below sample.
```
files": [
            {
                "hash": "sha256:4bda5343742c17873fcc68c6172ff4e426b1cbd6b2aacad4eec36664b928756e",
                "path": "data.rds",
                "size": 97
            },
            {
                "hash": "sha256:501aa01ab4176a4afd517a8c132429b12bc2ad3eb3430487eec0c5453305eef5",
                "path": "orderly.R",
                "size": 103
            }
        ]
```

shouldn't this be a nested array where a packet contains download heading and array of files as seen below in the old orderly?

<img width="994" alt="Screenshot 2023-07-14 at 13 29 59" src="https://github.com/mrc-ide/packit/assets/4240748/9344e10c-fc53-4cbf-8cd1-56721bb456df">


I am not sure we are returning all the list of files but this is what the endpoint provided. Kindly pre-review to give pointers.
